website/docs/r/vrf.html.markdown

```diff
@@ -49,6 +58,8 @@ resource "aci_vrf" "foovrf" {
 * `relation_fv_rs_ctx_to_ext_route_tag_pol` - (Optional) Relation to class l3extRouteTagPol. Cardinality - N_TO_ONE. Type - String.
 * `relation_fv_rs_ctx_to_bgp_ctx_af_pol` - (Optional) Relation to class bgpCtxAfPol. Cardinality - N_TO_M. Type - [Set of Map].
 
+Note: In the APIC GUI,a VRF (fvCtx) is also called a "Context"or "PrivateNetwork."
```

Comment:
I would use might/was called as we have almost changed it everywhere now. (Or should have)